pyaedt/application/Variables.py

```diff
@@ -986,7 +1112,8 @@ def __mul__(self, other):
         >>> assert result_2.numeric_value == result_1.numeric_value
         >>> assert result_2.unit_system == result_1.unit_system
 
-        Multiply 'Voltage' times 'Current' to obtain 'Power'
+        Multiply '`Voltage`' times '`Current'` to obtain '`Power'`.
```

Comment:
```suggestion
        Multiply voltage times current to obtain power.
```
I think we can just write this out here.